my-app/src/favorites.js

```diff
@@ -0,0 +1,69 @@
+import React, {Fragment, useState} from 'react'
+
+const Contents = () => {
+  const [data] = useState([
+    {
+      images: '/img/dubstep2.jpg',
+      title: 'Living Nightmare',
+      discaption: 'Snowflake',
+      time: '2:58'
+    },
+    {
+      images: '/img/happyrock.jpg',
+      title: 'Dubstep',
+      discaption: 'Bensound',
+      time: '2:04'
+    },
+    {
+      images: '/img/retrosoul.jpg',
+      title: '42',
+      discaption: 'Secret Of Life',
+      time: '1:18'
+    },
+    {
+      images: '/img/snowflake.jpg',
+      title: 'Retrosoul',
+      discaption: 'Bensound',
+      time: '2:55'
+    },
+    {
+      images: '/img/snowflake2.jpg',
+      title: 'Holiday Funky Blues',
+      discaption: 'Snowflake',
+      time: '2:20'
+    },
+    {
+      images: '/img/track3.jpeg',
+      title: 'Happy Rock',
+      discaption: 'Bendsound',
+      time: '12:00',
+      
+    },
```

Comment:
```suggestion 
    }
```

Xóa phẩy cuối trong 1 array đi